### PR TITLE
[zh] Fix typos in IPv4/IPv6 Dual Stack concept (IPv4/IPv6 双协议栈)

### PR DESCRIPTION
**Pull request by YunWZ**
Correct typos

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [zh] Fix typos in IPv4/IPv6 Dual Stack concept (IPv4/IPv6 双协议栈)

File: content/zh-cn/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -2,12 +2,21 @@
 title: 网络策略
 content_type: concept
 weight: 50
+description: >-
+  如果您想在 IP 地址或端口级别（OSI 第 3 层或第 4 层）控制流量，
+  NetworkPolicies 允许您为集群内以及 Pod 和外部世界之间的流量指定规则。 
+  您的集群必须使用支持 NetworkPolicy 实施的网络插件。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/network-policies.md:8]:
```suggestion
  如果你想在 IP 地址或端口级别（OSI 第 3 层或第 4 层）控制流量，
  NetworkPolicies 允许你为集群内以及 Pod 和外部世界之间的流量指定规则。 
  你的集群必须使用支持 NetworkPolicy 实施的网络插件。
```

**Review by windsonsea — COMMENTED** [review]
You may need to check the `still in sync` file and find whether any inconsistency with en:
```
content/zh-cn/docs/concepts/services-networking/dual-stack.md
```

File: content/zh-cn/docs/concepts/services-networking/network-policies.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/services-networking/network-policies.md:8]: NetworkPolicies 可以改为单数

Comment (YunWZ) [reply content/zh-cn/docs/concepts/services-networking/network-policies.md:8]: OK，i will do it later。

File: content/zh-cn/docs/concepts/services-networking/dual-stack.md [hunk 2]
```diff
@@ -247,11 +247,11 @@ These examples demonstrate the behavior of various dual-stack Service configurat
    this Service, Kubernetes assigns a cluster IP for the Service from the first configured
    `service-cluster-ip-range` and sets the `.spec.ipFamilyPolicy` to `SingleStack`. ([Services
    without selectors](/docs/concepts/services-networking/service/#services-without-selectors) and
-   [headless Services](/docs/concepts/services-networking/service/#headless-services) with selectors 
+   [headless Services](/docs/concepts/services-networking/service/#headless-services) with selectors
    will behave in this same way.)
 -->
 1. 此服务规约中没有显式设定 `.spec.ipFamilyPolicy`。当你创建此服务时，Kubernetes
-   从所配置的第一个 `service-cluster-ip-range` 种为服务分配一个集群IP，并设置
+   从所配置的第一个 `service-cluster-ip-range` 中为服务分配一个集群IP，并设置
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/dual-stack.md:254]:
```suggestion
   从所配置的第一个 `service-cluster-ip-range` 中为服务分配一个集群 IP，并设置
```

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by Sea-n** [comment]
Please also include changes from PR #37494.

**Comment by YunWZ** [comment]
> Please also include changes from PR #37494.

done.

**Review by Sea-n — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**